package.json

```diff
@@ -13,7 +13,7 @@
     "generate:config:local": "env-cmd foundation/environment/security.env.local npm run generate:config",
     "generate:config:prod": "env-cmd foundation/environment/security.env.prod npm run generate:config",
     "update-schema": "babel-node foundation/graphql/updateSchema.js",
-    "build:frontend": "NODE_ENV=production env-cmd foundation/environment/security.env.prod webpack --config foundation/webpack/webpack.prod.js -p && cp ./app/favicon.ico ./build/frontend/favicon.ico",
+    "build:frontend": "NODE_ENV=production env-cmd foundation/environment/security.env.prod webpack --progress --config foundation/webpack/webpack.prod.js -p && cp ./app/favicon.ico ./build/frontend/favicon.ico",
```

Comment:
This progress flag adds a bit more context to the webpack build rather than it hanging for a few seconds until it is finished. It can be moved to the webpack config, live here as a CLI flag, or completely removed. I'm fine with whatever y'all decide.